filter.imp

```diff
@@ -1,13 +1,111 @@
 [
-   { include: ["@.asm-generic/unaligned.h.", private, "<asm/unaligned.h>", "public"] },
+   # The following asm-generic files will always be replaced by asm files
+   # This list was generated from include/uapi/asm-generic/Kbuild
+   { include: ["@.asm-generic/auxvec.h.", private, "<asm/auxvec.h>", "public"] },
+   # bit twiddling operations should use linux/bits.h
+   { include: ["@.asm-generic/bitsperlong.h.", private, "<linux/bits.h>", "public"] },
+   { include: ["@.asm-generic/bpf_perf_event.h.", private, "<asm/bpf_perf_event.h>", "public"] },
+   { include: ["@.asm-generic/byteorder.h.", private, "<asm/byteorder.h>", "public"] },
    { include: ["@.asm-generic/errno.h.", private, "<linux/errno.h>", "public"] },
-   { include: ["@.asm-generic/errno-base.h.", private, "<linux/errno.h>", "public"] },
    { include: ["@.asm/errno.h.", private, "<linux/errno.h>", "public"] },
```

Comment:
For this table, probably none of the public headers should be linux/*.h, but asm/*.h.

Then below, you can remap asm/*.h to linux/*.h with a comment.